pkg/sdk/simpleschema.go

```diff
@@ -0,0 +1,115 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package sdk
+
+import (
+	"fmt"
+
+	awssdkmodel "github.com/aws-controllers-k8s/code-generator/pkg/api"
+	ackmodel "github.com/aws-controllers-k8s/code-generator/pkg/model"
+	simpleschema "github.com/kro-run/kro/pkg/simpleschema"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
nit: need to order the imports: checkout the standard here https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e